src/main/java/org/jboss/forge/arquillian/command/algeron/consumer/PactAlgeronConsumerTestSetup.java

```diff
@@ -33,6 +33,7 @@ public JavaClassSource updateTest(JavaClassSource test, String consumer, String
         annotateTestClass(test, consumer, provider);
         createContractFragment(test, fragmentName);
         createTestMethod(test, consumer, provider, fragmentName);
+        removeTestMethod("should_be_deployed", test);
```

Comment:
We can run test as client by using deployment=false which is already there in https://github.com/forge/arquillian-addon/blob/master/src/main/resources/JUnitTest.ftl
So this is standalone mode(client mode). So we can cover this using existing template.

but for cube or algeron we don't need this, then I think it's good to remove from standalone template.